payload/reggae/dub/info.d

```diff
@@ -168,8 +168,22 @@ struct DubInfo {
         else
             static assert(false, "Unknown compiler");
 
+        const(string)[] pkgDflags = dubPackage.dflags;
+        version(LDC) {
```

Comment:
Shouldn't this check if the D compiler is ldc instead?

Reply:
I've just used the `version` you use a few lines up. [FWIW, it'd be only required for the ldc2 driver though, not ldmd2.]